test/www/jxcore/test/thalicryptomanagertests.js

```diff
@@ -0,0 +1,61 @@
+require('./mockmobilegetdocpath');
+var test = require('tape');
+var cryptomanager = require('../thali/thalicryptomanager');
+
+
+test('successfully create a new pkcs12 file and return hash value', function (t) {
+  var errorMessage = null,
+      fileLocation = './pkcs12foldergood';
```

Comment:
You are assuming this location exists which it generally shouldn't since we have nothing to put in there and git doesn't record directories. Instead you should include a global setup using tape and just have it create the directory. Also please make sure to put in a global cleanup and have it delete the directory and its contents. You can use fs-extras-promise to handle the recursive delete.